### PR TITLE
executor: support to update log level by SQL

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve? 
Support to update log level by SQL.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: sessionctx/variable/sysvar.go [hunk 1]
```diff
@@ -707,6 +707,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeSession, TiDBSlowLogThreshold, strconv.Itoa(logutil.DefaultSlowThreshold)},
 	{ScopeSession, TiDBRecordPlanInSlowLog, strconv.Itoa(logutil.DefaultRecordPlanInSlowLog)},
 	{ScopeSession, TiDBEnableSlowLog, strconv.Itoa(logutil.DefaultTiDBEnableSlowLog)},
+	{ScopeSession, TiDBLogLevel, logutil.DefaultLogLevel},
```
Comment (SunRunAway) [inline sessionctx/variable/sysvar.go]: Since this viriable affects a global setting, could we only support `ScopeGlobal`?

Comment (qw4990) [reply sessionctx/variable/sysvar.go]: Using `ScopeGlobal` makes the config item visible to all TiDB instances in the same cluster by storing this config item to TiKV and other TiDB instances will load it later.
Actually this config item should be `ScopeInstance` (or `ScopeServer`), but now this scope level is not supported by TiDB, which is a known issue.
Some other config items like `TiDBSlowLogThreshold`, `TiDBRecordPlanInSlowLog` also have this problem...

Comment (lonng) [reply sessionctx/variable/sysvar.go]: Seems this function is duplicated with `update ... cluster_config`?

Comment (qw4990) [reply sessionctx/variable/sysvar.go]: Is there any document about this feature? `update ... cluster_config`? @lonng

Comment (qw4990) [reply sessionctx/variable/sysvar.go]: And I will discuss with our PM about this config item's scope level.

**Pull request closed without merging**